### PR TITLE
Allow to establish a connection parallel and serially

### LINKED ISSUE #3: Make the load distributable
Add a new CLI flag. If it's set, connect serially, not in parallel.

### PR DESCRIPTION
**Pull request by yhabteab**
resolves #3

### CODE REVIEW

**Review by Al2Klimov — COMMENTED** [review]
Why not just `s/go ProcessOutgoingConnection/ProcessOutgoingConnection/` if the flag is not set?

File: main.go [hunk 1]
```diff
@@ -105,7 +106,11 @@ func main()  {
 	Addr := fmt.Sprintf("%s%s%d", *host, ":", *port)
 
 	for i := 0; i < *concurrency; i++ {
-		go ProcessOutgoingConnection(Addr, tlsConfigs[i], sleep, message)
+		if *parallel == true {
```
Comment (Al2Klimov) [inline main.go]: Why ` == true`?

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (yhabteab) [reply main.go]: What do you mean? How do you compare boolean flags?

Comment (Al2Klimov) [reply main.go]: I don't **compare** them. I **check** them. Just remove the mentioned characters from the code and you'll see what I mean.

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**